Source/ACE/Managers/GuidManager.cs

```diff
@@ -164,5 +169,23 @@ public static ObjectGuid NewItemGuid()
         {
             return new ObjectGuid(itemAlloc.Alloc());
         }
+
+        /// <summary>
+        /// Returns Most Recently assigned Guid for Items / Player Items, to be used only for informational purposes.
+        /// </summary>
+        /// <returns></returns>
+        public static ObjectGuid CurrentItemGuid()
```

Comment:
And I'm not a fan of the "idlist" functions either.  As I commented earlier, they are not implemented correctly and of questionable value.